src/canopy_server/app.py

```diff
@@ -95,13 +95,19 @@
 logger: logging.Logger
 
 
```

Comment:
@izellevy you also need to add `/{namespace}/` param to the `/upsert` and `/query` routes.  
As we discussed - for those two routes it can be at the end, no need to add another dedicated route (e.g. `/query/{namespace}`)

Reply:
I think it is better to keep the same convention from the users perspective. So I prepended the namespace for now. Let's take it offline